### DISCUSSION TITLE
Linux port of gui version?

### DISCUSSION THREAD

**Discussion by Moody1021** [hunar4321/particle-life#88] [General]
Is there a Linux version of particle-life planned? It looks like the openmp and openframeworks are both available on Linux.
Thanks,
Moody

**Comment by ker2x** [comment]
it should work as-is on linux. you're asking for a binary ?

**Reply by Moody1021** [reply]
The python particle.py works fine on Linux. The html versions 2d and 3d also work fine. However, I don't see a gui based version for Linux where I can change the rules, etc. I looked under the src directory. I see the main.cpp, etc. However, there are a bunch of dlls under the bin directory. I presume those are necessary to compile under windows. However, I don't see the source for those dlls on github. I seem to be missing something obvious from what your question implies. What do I need to do compile and run the .cpp files?
Thanks!
